src/js/components/Header/stories/Responsive.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Grommet, Header, Anchor, Box, ResponsiveContext, Menu } from 'grommet';
+import { Grommet as GrommetIcon, Menu as MenuIcon } from 'grommet-icons';
+import { grommet } from 'grommet/themes';
+
+const Responsive = () => (
+  <Grommet theme={grommet}>
+    <Header background="light-4" pad="medium" height="xsmall">
+      <Anchor
+        href="https://tools.grommet.io/"
+        icon={<GrommetIcon color="brand" />}
+        label="Grommet Tools"
+      />
+      <ResponsiveContext.Consumer>
+        {size =>
+          size === 'small' ? (
+            <Box justify="end">
+              <Menu
+                a11yTitle="Navigation Menu"
+                dropProps={{ align: { top: 'bottom', right: 'right' } }}
+                icon={<MenuIcon color="brand" />}
+                items={[
+                  {
+                    label: <Box pad="small">Grommet Tools </Box>,
+                    href: 'https://tools.grommet.io/',
+                  },
```

Comment:
Not sure why we need this entry? It doesn't exist on the large screen option where you have only two links.

Reply:
Removed in latest commit